scripts/dev_cli.sh

```diff
@@ -278,7 +289,8 @@ cmd_tests() {
     integration_windows=false
     libc="gnu"
     arg_vols=""
-
+    hypervisor="kvm"
+    cp_args=("$@")
```

Comment:
I think `dup_args` or just `saved_args` is clearer.

I can see why you want to create it here because you want to preserve `--hypervsior`, but you've subtly changed what is passed to test scripts because you also pass all other arguments.